src/expressions.rst

```diff
@@ -4813,3 +4884,25 @@ within the :t:`capturing expression`, as follows:
 :dp:`fls_wvob7114tfat`
 A tool selects the first :t:`capture mode` that is compatible with the use of
 the :t:`capture target`.
+
+.. _fls_ZfIBiJMf8qE1:
+
+Arithmetic Overflow
+-------------------
+
+:dp:`fls_oFIRXBPXu6Zv`
+An :dt:`arithmetic overflow` occurs if an :t:`arithmetic expression` or a
```

Comment:
"Arithmetic overflow" should be defined in the Glossary. The sentence should also be left here as an introduction, but the reference to "arithmetic overflow" should be changed to :t:.

Reply:
Is there any way to link from the glossary to this paragraph? That might be more convenient for finding it

Reply:
Unfortunately no. I raised this issue in the appropriate Zulip channel. Whatever the solution, it would encompass the whole Glossary. In the mean time, you should still define `arithmetic overflow` in the Glossary.

Reply:
Okay, did that